README.md

```diff
@@ -1 +1,32 @@
-# Portfolio
\ No newline at end of file
+# My Portfolio
+
+> This is my portfolio describing what I am capable of doing and what value I can bring in a team.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)

```

Comment:
- [ ] [REQUIRED] Please, can you add the live link for the live versions. _🔗_. This would help potential employers and recruiters see how the real website looks 💅